stages/vault/sharedsvc/vault-policies/lead-aws-developer.hcl

```diff
@@ -1,3 +1,9 @@
+// read access to a specific secret
+
+path "lead/data/aws/{{identity.entity.aliases.${mount_accessor}.metadata.account_id}}/*" {
+    capabilities = ["read"]
+}
```

Comment:
This was needed for the `Developer` role to actually use `vault_generic_secret` data sources within Terraform